src/teleop/gui/src/components/AutonTask.vue

```diff
@@ -94,12 +88,25 @@ export default {
       messageType : 'sensor_msgs/NavSatFix'
     });
 
-    this.localization_sub = new ROSLIB.Topic({
+    this.tfClient = new ROSLIB.TFClient({
       ros : this.$ros,
-      name : '/imu/data',
-      messageType : 'sensor_msgs/Imu'
+      fixedFrame : 'odom',
+      angularThres : 0.01,
+      transThres : 0.01
```

Comment:
Can you add a comment or something for what these mean? I'm guessing maybe the thresholds for the subscription right below this to get called